src/components/UI/common/ProcessStep.tsx

```diff
@@ -0,0 +1,23 @@
+import { Grid, GridItem } from '@chakra-ui/react';
+import { FC } from 'react';
+
+import { PageText, StepArrow, StepHeader } from '../';
+
+interface Props {
+  title: string;
+  withArrow?: boolean;
+}
+
+export const ProcessStep: FC<Props> = ({ title, withArrow, children }) => {
```

Comment:
Not a blocker for this PR: is it worth to consider `withArrow` as `true` by default? since most of them will require to have the arrow.

Reply:
Makes sense. I've updated `ProcessStep` to display the arrow by default, unless `isLastStep` is set to true.